client/src/components/answers/TopAnswer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+
+const TopAnswer = props => {
```

Comment:
Is there a setting in your editor to "use spaces rather than tabs"? Maybe this: https://stackoverflow.com/questions/22608160/atom-change-indentation-mode